lucetc/tests/wasm.rs

```diff
@@ -370,32 +370,6 @@ mod module_data {
 
     // XXX adding more negative tests like the one above is valuable - lets do it
 
-    #[test]
-    fn invalid_module() {
-        use lucetc::Error as LucetcError;
-        use std::fs::File;
-        use std::io::Read;
-        // I used the `wast2json` tool to produce the file invalid.wasm from an assert_invalid part
-        // of a spectest (call.wast)
-        let wasmfile = PathBuf::from("tests/wasm/invalid.wasm");
-        let mut m = Vec::new();
-        let mut file = File::open(&wasmfile).expect("open module file");
-        file.read_to_end(&mut m).expect("read contents of module");
-
-        let b = Bindings::empty();
-        let builder = Compiler::builder();
-        let c = builder.create(&m, &b);
-        assert!(
-            c.is_err(),
-            "compilation error because wasm module is invalid"
```

Comment:
FWIW a crate like `wat` makes it easy to make wasm modules that are invalid at least at the type level (binary level is a bit more difficult), but the new test here seems fine either way.

Reply:
I didn't actually try using `wat` because my intuition was that it would reject creating a binary of an improperly typed module. This test is a bit more janky than I usually like, but I wasn't able to come up with a better idea.